managed/services/telemetry/config.default.yml

```diff
@@ -24,6 +24,15 @@ telemetry:
       - metric_name: "pmm_server_backup_management_enabled"
         column: "backup_management_enabled"
 
+  - id: PMMServerAlertManagerUsed
+    source: PMMDB_SELECT
+    query: (CASE a WHEN '""' THEN 'false' ELSE 'true' END) as alert_manager_set FROM settings s, jsonb_extract_path(s.settings, 'alert_manager_url') AS a
```

Comment:
`AS a` ? Maybe to use something more descriptive?

Reply:
@dliakhov would you mind changing it to `alert_manager_url`

Reply:
Yah, makes sense

refactored that